README.md

```diff
@@ -21,13 +21,13 @@ python src/download_data.py download_data.py --url=https://github.com/invinst/ch
 python src/download_data.py download_data.py --url=https://github.com/invinst/chicago-police-data/blob/master/data/unified_data/profiles/final-profiles.csv.gz?raw=true --path=data/raw/demographics.csv
 
 # Clean / preprocess data
-Rscript src/read_preprocess_data.R --file_path=data --out_dir=data/processed
+Rscript src/read_preprocess_data.R --file_path=data/raw --out_dir=data/processed
 
 # Create exploritory data analysis figures and write to file
-Rscript src/generate_EDA_figures.R --all_data=data/complaints_all_staff --police_data=data/complaints_police --out_dir=eda/images
+Rscript src/generate_EDA_figures.R --all_data=data/processed/complaints_all_staff --police_data=data/processed/complaints_police --out_dir=eda/images
```

Comment:
Sorry, the data should refer to the actual csv file I fixed it below. Also added the ../ because this starts at the src file. Not 100% on this since I'm not testing it out, but I believe so...

`Rscript src/generate_EDA_figures.R --all_data=../data/processed/complaints_all_staff.csv --police_data=../data/processed/complaints_police.csv --out_dir=../eda/images`

Reply:
Hi Micah, 

Good catch with the .csv I missed those 😅 

I think based on how at @ebs33 had set this up, we are started at the root directory since we are pointing at the script  in the src directory (i.e. src/generate_EDA_figures.R ) so the ../data is not needed.

Reply:
Oh you're right. I didn't see that, we should be good then! just the .csv part is all then